airbyte-config/persistence/src/main/java/io/airbyte/config/persistence/ConfigRepository.java

```diff
@@ -138,6 +142,14 @@ public void writeStandardSourceDefinition(final StandardSourceDefinition sourceD
         sourceDefinition);
   }
 
+  public void deleteStandardSourceDefinition(final UUID sourceDefId) throws IOException {
```

Comment:
is it overkill to have tests for this?

Reply:
i kinda think so. we have generally not written tests for this class, except for handling weird cases (e.g. tombstones)